cmd/sectigo/main.go

```diff
@@ -3,7 +3,6 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
```

Comment:
Most of the changes in this file are an upgrade to urifave v2 and the addition of the `serve` command to serve the fake Sectigo service.